experiments/Atmos_LES/dycoms.jl

```diff
@@ -15,6 +16,9 @@ using CLIMA.MoistThermodynamics
 using CLIMA.PlanetParameters
 using CLIMA.VariableTemplates
 
+const clima_dir = dirname(Base.active_project())
```

Comment:
If you want the path to `CLIMA/src` you probably want `dirname(pathof(CLIMA))`

```
 julia> using BenchmarkTools
[ Info: Precompiling BenchmarkTools [6e4b80f9-dd63-53aa-95a3-0cdb28fa8baf]

julia> pathof(BenchmarkTools)
"/home/vchuravy/src/BenchmarkTools/src/BenchmarkTools.jl"

julia> dirname(pathof(BenchmarkTools))
"/home/vchuravy/src/BenchmarkTools/src"
```

Reply:
Nice, I didn't know of `pathof`. Perhaps that's more what I need (I actually need `CLIMA/`, though).